documentation/cobigen-core_configuration.asciidoc

```diff
@@ -374,18 +380,187 @@ _(Since version 4.1.0)_  An special case of `<incrementRef ...>` is the external
 
 However, if we want to reference an increment that it is not defined inside our `templates.xml` (an increment defined for another trigger), then we can use external `incrementRef` as shown below:
 
-```xml 
+```xml
 <increment name="..." description="...">
     <incrementRef ref="trigger_id::increment_id"/>
 </increment>
 ```
 
 The ref string is split using as delimiter `::`. The first part of the string, is the `trigger_id` to reference. That trigger contains an `increment_id`. Currently, this functionality only works when both templates use the same kind of input file.
 
+
+== Template-set Configuration
+
+
+With version 6.0 you are working with template-sets rather than the old monolithic structure with template.xml and context.xml files.
```

Comment:
Template Set version 1.0 or we should mention the new CobiGen release version here instead.